pkgs/applications/window-managers/sway/contrib.nix

```diff
@@ -0,0 +1,98 @@
+{ stdenv
+
+, fetchurl
+, coreutils
+, makeWrapper
+, sway-unwrapped
+, wl-clipboard
+, libnotify
+, slurp
+, grim
+, jq
+
+, python3Packages
+}:
+
+{
+
+grimshot = stdenv.mkDerivation rec {
+  pname = "grimshot";
+  version = "2020-05-08";
+  rev = "b1d08db5f5112ab562f89564825e3e791b0682c4";
+
+  # master has new fixes and features, and a man page
+  # after sway-1.5 these may be switched to sway-unwrapped.src
+  bsrc = fetchurl {
+    url = "https://raw.githubusercontent.com/swaywm/sway/${rev}/contrib/grimshot";
+    sha256 = "1awzmzkib8a7q5s78xyh8za03lplqfpbasqp3lidqqmjqs882jq9";
+  };
+
+  msrc = fetchurl {
+    url = "https://raw.githubusercontent.com/swaywm/sway/${rev}/contrib/grimshot.1";
+    sha256 = "191xxjfhf61gkxl3b0f694h0nrwd7vfnyp5afk8snhhr6q7ia4jz";
+  };
+
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  phases = [ "installPhase" "fixupPhase" "checkPhase" ];
+
+  installPhase = ''
+    install -D ${msrc} $out/share/man/man1/grimshot.1
+
+    install -Dm 0755 ${bsrc} $out/bin/grimshot
+    wrapProgram $out/bin/grimshot --set PATH \
+      "${stdenv.lib.makeBinPath [
+        sway-unwrapped
+        wl-clipboard
+        coreutils
+        libnotify
+        slurp
+        grim
+        jq
+        ] }"
+  '';
+
+  doCheck = true;
+
+  checkPhase = ''
+    # check always returns 0?
+    if [[ $($out/bin/grimshot check | grep "NOT FOUND") ]]; then false; fi
```

Comment:
Is this an open TODO? (Questionmark in the comment)

Reply:
it's a note of uncertainty about whether `grimshot check` always returns 0, which it appears to do

Reply:
Yes, it should always return 0: https://github.com/swaywm/sway/blob/master/contrib/grimshot#L111 (the last command before is [echo](https://github.com/swaywm/sway/blob/master/contrib/grimshot#L87) -> always zero).

Reply:
removed the questionmark